modin/pandas/test/test_io.py

```diff
@@ -835,20 +808,12 @@ def test_read_csv_quoting(
             )
 
     # Error Handling parameters tests
-    @pytest.mark.parametrize("warn_bad_lines", [True, False, None])
-    @pytest.mark.parametrize("error_bad_lines", [True, False, None])
+    @pytest.mark.skip
```

Comment:
why do we skip this test?

Reply:
The reason is unknown, I can not reproduce locally. Let's fix it separately.

Reply:
At least make an umbrella issue and reference it in all places where tests started to fail after migration to newer pandas so you had to disable them. Then we can split the issue by root causes and create new ones when something is known better, but I believe there should exist at least one tracker.

Reply:
made an umbrella issue